src/main/resources/public/alto-boot/newsession.html

```diff
@@ -147,59 +142,31 @@
 </script>
 </head>
 <body>
-	<div class="topbar">
-		<div class="fill">
-			<div class="container">
-				<a class="brand" href="#">Interactive Document Labeling</a>
-			</div>
-		</div>
-	</div>
-	<div class="container">
-
-		<div class="content">
-			<!--div class="page-header">
-	          <h1>Page name <small>Supporting text or tagline</small></h1>
-	        </div-->
-			<div class="row">
-				<div class="span16">
-					<h2>New Document Labeling Session</h2>
-
-					<form name="new_session_form" method="GET" action="ui.html">
-
-						<div class="clearfix">
-							<label for="username">ID</label>
-							<div class="input">
-								<input class="xlarge" id="username" name="username" size="30"
-									type="text" />
-							</div>
-						</div>
-						
-						<div class="clearfix">
-							<label for="studycondition">Study Condition</label>
-							<div class="input">
-								<select name="studycondition" id="studycondition">
-									<option value="TA">ALTO (TA)</option>
-									<option value="LR">LR</option>
-									<option value="LA">LA</option>
-									<option value="TR">TR</option>
-								</select>
-							</div>
-						</div>
-						<div class="actions">
-							<input id="start-new-session" type="submit"
-								class="btn large primary" value="Start">
-						</div>
-						</fieldset>
-					</form>
+<nav class="navbar navbar-dark bg-dark">
+	<a class="navbar-brand" href="#">Interactive Document Labeling</a>
+</nav>
+<div class="container">
+	<div class="card mt-3">
+		<h4 class="card-header">New Document Labeling Session</h4>
+		<div class="card-body">
+			<form name="new_session_form" method="GET" action="ui.html">
+				<div class="form-group">
+					<label class="col-form-label" for="username">Username</label>
+					<input type="text" class="form-control" id="username" name="username" placeholder="Username" required>
```

Comment:
also added a `required` here to prevent the user from submitting the form without a username. i expect that's the desired behavior?